src/metabase/legacy_mbql/util.cljc

```diff
@@ -849,6 +849,16 @@
            [:field (id :guard integer?) opts]
            [id (:source-field opts)]))))
 
+(defn pred-matches-form?
+  "Check if `form` or any of its children forms match `pred`. Absolute most predicates will not match (this function is
```

Comment:
I'm struggling to parse what "Absolute most" means. I think you can just say "Most"?

Reply:
I'll reword.